### ISSUE TITLE
Update KB

### ISSUE THREAD

**Issue by themkdemiiir** [WGLab/Phen2Gene#32]
Could you provide a status report on this matter? Additionally, I'd like to revise the Knowledge Base. If you have any scripts for modifying Phenolyzer outputs corresponding to each HPO term, I would be happy to use them. I appreciate your help.

**Comment by kaichop** [comment]
I will try to complete this update on Phen2Gene. The knowledgebase is actually already updated in Phenolyzer, so in principle Phen2gene can be directly updated using phenolyzer but I am not sure if the results are reliable without testing. So I never got the time to really do it. (I also think your question is about generate scores for HPO from phenolyzer; there is no special script that is needed, you can directly run phenolyzer and get the output).

**Comment by themkdemiiir** [comment]
I appreciate your clarification. I was asking about creating the knowledge base, which is derived from the output of Phenolyzer and then processed, correct? How did you process these phenolyzer outputs for each HPO term? Could ı use phenolyzer output on phen2gene directly?

**Comment by kaichop** [comment]
Yes, the phenolyzer output is fed to phen2gene (but through a pre-computed
means so that phen2gene is fast to run) so in principle you could actually
directly use phenolyzer output on phen2gene for these specific HPO terms
that you are interested in. I did update the knowledegebase for phenolyzer
a few weeks back, but need some time to pre-compute scores for all updated
HPO terms, then use them in phen2gene (i.e. the knowledgebase for phen2gene
is not updated yet).
